pilot/pkg/security/authn/utils/utils.go

```diff
@@ -75,9 +75,9 @@ func BuildInboundTLS(mTLSMode model.MutualTLSMode, node *model.Proxy,
 		ciphers = mc.MeshMTLS.CipherSuites
 	}
 	if ciphers == nil {
-		ciphers = tls_features.TLSCipherSuites.Get()
-		if len(ciphers) == 0 {
-			ciphers = SupportedCiphers
+		opensslCiphers := tls_features.TLSCipherSuites.Get()
+		if len(opensslCiphers) > 0 {
```

Comment:
I don't think we need the if-clause anymore

Reply:
Envoy documentation does not clarify what means "unset", so `nil` is safer than an empty slice.